assets/app/scripts/services/builds.js

```diff
@@ -73,7 +74,8 @@ angular.module("openshiftConsole")
             $scope.alerts["rebuild"] = 
             {
               type: "success",
-              message: "Build " + buildName + " is being rebuilt as " + build.metadata.name + "."
+              message: "Build " + buildName + " is being rebuilt as " + build.metadata.name + ".",
+              link: "project/" + build.metadata.namespace + "/browse/builds/" + buildConfigName +"/" + build.metadata.name
```

Comment:
you dont need buildConfigName at all if you use navigateResourceURL, it will look for the build config on the builds labels if it exists and build the appropriate URL either way.  Since that is all that buildConfigName was being used for take it out everywhere that it was added.